src/Analysis/Ast/Impl/Specializations/Typing/Values/TypingTuple.cs

```diff
@@ -17,6 +17,7 @@
 using Microsoft.Python.Analysis.Types;
 using Microsoft.Python.Analysis.Values;
 using Microsoft.Python.Analysis.Values.Collections;
+using System.Collections.Generic;
```

Comment:
Make sure your usings are getting sorted; System usings should automatically be getting put above any others.

Reply:
Ok!